PSICover/PSICover.cs

```diff
@@ -183,6 +228,52 @@ class Analyzer {
          html = html.Replace ("\u00ab", "&lt;").Replace ("\u00bb", "&gt;");
          File.WriteAllText (htmlfile, html);
       }
+      var cSum = mSum.OrderBy (a => a.Percent).ToList ();

```

Comment:
cSum does not follow our style. Only count should start with c. Also, ToList is not needed. You could use foreach in 235 line.